public/index.html

```diff
@@ -10,34 +10,14 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
- [ ] In your readme file, it seems like your MIT license link is broken. Kindly connect your MIT license with the correct link. Thank you!

Reply:
Thank you for your feedback. I have resolved this issue. 

[MIT LICENSE](https://github.com/kennedyowusu/bookstore/blob/develop/LICENSE.md))